internal/cli/projects/update.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package projects
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/file"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20231115008/admin"
+)
+
+const updateTemplate = "Project '{{.Id}}' updated.\n"
+
+type UpdateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	filename string
+	fs       afero.Fs
+	store    store.ProjectUpdater
+}
+
+func (opts *UpdateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *UpdateOpts) Run() error {
+	params, err := opts.newUpdateProjectParams()
+	if err != nil {
+		return err
+	}
+
+	r, err := opts.store.UpdateProject(params)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *UpdateOpts) newUpdateProjectParams() (*atlasv2.UpdateProjectApiParams, error) {
+	groupUpdate := &atlasv2.GroupUpdate{}
+	if err := file.Load(opts.fs, opts.filename, groupUpdate); err != nil {
+		return nil, err
+	}
+
+	return &atlasv2.UpdateProjectApiParams{
+		GroupId: opts.ConfigProjectID(), GroupUpdate: groupUpdate}, nil
```

Comment:
since it uses only the arg and not the config 
```suggestion
	return &atlasv2.UpdateProjectApiParams{
		GroupId: opts.ProjectID, GroupUpdate: groupUpdate}, nil
```